io.github.mrstickypiston.stickyhours.yml

```diff
@@ -0,0 +1,90 @@
+app-id: io.github.mrstickypiston.stickyhours
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: stickyhours
+cleanup:
+  - /include
+  - /share/man
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+modules:
+  - name: cpython
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin
+      - cp --archive python/bin/* /app/bin
+      - chmod -R +x /app/bin
+      - mkdir -p /app/include
+      - cp --archive python/include/* /app/include
+      - mkdir -p /app/lib
+      - cp --archive python/lib/* /app/lib
+      # Update installation prefix for pip package builds
+      - sed --follow-symlinks -i 's/prefix="\/install"/prefix="\/app"/g' /app/bin/python3-config
+      - sed --follow-symlinks -i 's/prefix=\/install/prefix=\/app/g' /app/lib/pkgconfig/python3.pc /app/lib/pkgconfig/python3-embed.pc
```

Comment:
don't use sed. Just fix the tarbal all at that point since you control it.

Reply:
i dont actually control that part of the tarbal as it is generated by briefcase

Reply:
and it doesn't know about `prefix`? Then maybe it's just the wrong tool.

Reply:
also why do you install python?

Reply:
uses sdk python now